test/test_benchmark.py

```diff
@@ -240,6 +267,16 @@ def get_output_files(self):
     return ret
 
 
+class SizeBenchmarker(EmscriptenBenchmarker):
```

Comment:
Maybe add a comment here about what this does, what the format it saves it in is, etc.? (or is that documented somewhere else?)